### PR TITLE
Update CSP Documentation

### PR DESCRIPTION
**Pull request by BenceSzalai**
The CSP documentation lacks some crucial information and has a confusing structure. These commits supposed to make that part of the docs more useful.

### CODE REVIEW

**Comment by clarkdo** [comment]
@pi0 I'll have a look, but @manniL is the main author of CSP 😄

**Review by clarkdo — APPROVED** [review]
Look good

File: content/en/api/configuration-render.md [hunk 1]
```diff
@@ -178,13 +178,28 @@ See [serve-static](https://www.npmjs.com/package/serve-static) docs for possible
...
   - Default: `false`
 
-> Use this to configure to load external resources of Content-Security-Policy
+> Use this to configure Content-Security-Policy to load external resources
 
-Note that CSP hashes will not be added if `script-src` policy contains `'unsafe-inline'`. This is due to browser ignoring `'unsafe-inline'` if hashes are present. Set option `unsafeInlineCompatibility` to `true` if you want both hashes and `'unsafe-inline'` for CSPv1 compatibility.
+**Prerequisites:**
 
-In order to add [`<meta http-equiv="Content-Security-Policy"/>`](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) with all the CSP policies you need to set `csp.addMeta` to `true`.
+These CSP settings are only effective when using Nuxt server to serve your SSR application. The Policies defined under `csp.policies` are added to the response `Content-Security-Policy` HTTP header.
```
Comment (manniL) [inline content/en/api/configuration-render.md:185]: Would probably suggest to directly reference the target here:

```suggestion
These CSP settings are only effective when using Nuxt with `target: 'server'` to serve your SSR application. The Policies defined under `csp.policies` are added to the response `Content-Security-Policy` HTTP header.
```

File: content/en/guides/configuration-glossary/configuration-render.md [hunk 2]
```diff
@@ -176,11 +176,26 @@ See [serve-static](https://www.npmjs.com/package/serve-static) docs for possible
 - Type: `Boolean` or `Object`
   - Default: `false`
 
-> Use this to configure to load external resources of Content-Security-Policy
+> Use this to configure Content-Security-Policy to load external resources
 
-Note that CSP hashes will not be added if `script-src` policy contains `'unsafe-inline'`. This is due to browser ignoring `'unsafe-inline'` if hashes are present. Set option `unsafeInlineCompatibility` to `true` if you want both hashes and `'unsafe-inline'` for CSPv1 compatibility.
+**Prerequisites:**
+
+These CSP settings are only effective when using Nuxt server to serve your SSR application. The Policies defined under `csp.policies` are added to the response `Content-Security-Policy` HTTP header.
```
Comment (manniL) [inline content/en/guides/configuration-glossary/configuration-render.md:183]: Same as above. 
```suggestion
These CSP settings are only effective when using Nuxt with `target: 'server'` to serve your SSR application. The Policies defined under `csp.policies` are added to the response `Content-Security-Policy` HTTP header.
```

**Review by manniL — CHANGES_REQUESTED** [review]
LGTM besides a small notation

**Review by manniL — APPROVED** [review]

**Comment by debs-obrien** [comment]
Thanks guys and thanks @BenceSzalai 😃 great work

**Pull request merged**